index.js

```diff
@@ -51,6 +53,7 @@ function compression (options) {
   // options
   var filter = opts.filter || shouldCompress
   var threshold = bytes.parse(opts.threshold)
+  var defaultEncoding = opts.defaultEncoding || 'identity'
```

Comment:
We should not use `defaultEncoding`, it is in `zlib`. Anything that creates conflict down the line in related places should be avoided.

IMO, we could use `enforceEncoding`. It creates more clarity on the fact that we are allowing an enforcement. The default behavior in content negotiation is more like _what's best for the client_.